packages/@ama-sdk/schematics/schematics/typescript/shell/index.spec.ts

```diff
@@ -0,0 +1,102 @@
+
+const baseFileList = [
+  '/CONTRIBUTING.md',
+  '/jest.config.js',
+  '/readme.md',
+  '/tsconfig.doc.json',
+  '/tsconfig.build.json',
+  '/tsconfig.json',
+  '/.commitlintrc.json',
+  '/.editorconfig',
+  '/.eslintignore',
+  '/.eslintrc.js',
+  '/.gitignore',
+  '/.renovaterc.json',
+  '/.swagger-codegen-ignore',
+  '/.swcrc',
+  '/.versionrc.json',
+  '/.yarnrc.yml',
+  '/package.json',
+  '/configs/specReporter.config.json',
+  '/configs/tsconfig.test.json',
+  '/scripts/clear-index.js',
+  '/scripts/files-pack.js',
+  '/scripts/override-readme.js',
+  '/scripts/restore-readme.js',
+  '/src/index.ts',
+  '/src/fixtures/jasmine/index.ts',
+  '/src/fixtures/jasmine/package.json',
+  '/src/fixtures/jest/index.ts',
+  '/src/fixtures/jest/package.json',
+  '/src/helpers/index.ts',
+  '/src/helpers/package.json',
+  '/src/models/enums.ts',
+  '/src/models/index.ts',
+  '/src/models/patterns.ts',
+  '/src/models/base/index.ts',
+  '/src/models/core/enums.ts',
+  '/src/models/core/index.ts',
+  '/src/models/core/patterns.ts',
+  '/src/models/custom/enums.ts',
+  '/src/models/custom/index.ts',
+  '/src/models/custom/patterns.ts',
+  '/src/spec/package.json',
+  '/src/spec/index.ts',
+  '/src/spec/mock-factory/index.ts',
+  '/testing/tsconfig.spec.json',
+  '/.husky/commit-msg',
+  '/.husky/pre-commit',
+  '/.husky/.gitignore',
+  '/tsconfigs/esm2020/tsconfig.jasmine.json',
+  '/tsconfigs/esm2020/tsconfig.jest.json',
+  '/tsconfigs/esm2020/tsconfig.json',
+  '/tsconfigs/esm2020/tsconfig.source.json',
+  '/tsconfigs/tsconfig.jasmine.json',
+  '/tsconfigs/tsconfig.jest.json',
+  '/tsconfigs/tsconfig.source.json',
+  '/typedoc.json'
+];
+
+describe('Typescript Shell Generator', () => {
+  it('should generate basic SDK package', async () => {
+    const runner = new SchematicTestRunner('@ama-sdk/schematics', collectionPath);
+    const tree = await runner.runSchematic('typescript-shell', {
+      name: 'test-sdk',
+      package: 'sdk',
+      hosting: Hosting.OTHER
+    }, Tree.empty());
+
+    expect(tree.files.sort()).toEqual(baseFileList.sort());
+    expect(tree.files).not.toContain('/azure-pipelines.yml');
+  });
+
+  it('should generate correct package name', async () => {
+    const runner = new SchematicTestRunner('@ama-sdk/schematics', collectionPath);
+    const tree = await runner.runSchematic('typescript-shell', {
+      name: 'test-scope',
+      package: 'test-sdk',
+      hosting: Hosting.OTHER
+    }, Tree.empty());
+
+    const {name} = JSON.parse(tree.readContent('/package.json'));
+
+    expect(name).toEqual('@test-scope/test-sdk');
+  });
+
+  it('should generate azure templates', async () => {
+    const runner = new SchematicTestRunner('@ama-sdk/schematics', collectionPath);
+    const tree = await runner.runSchematic('typescript-shell', {
+      name: 'test-scope',
+      package: 'test-sdk',
+      hosting: Hosting.AZURE
+    }, Tree.empty());
+
+    expect(tree.files).toContain('/azure-pipelines.yml');
```

Comment:
you could also make sure the other files were generated

Reply:
I think the purpose of the test is to check that we have an azure pipeline generated 
we don't really care about these subfiles
the one taking into account to run a pipeline is this one only